src/phases/HelloWorld/ExamplePhase2.yml

```diff
@@ -1,5 +1,7 @@
 SchemaVersion: 2018-07-01
-Owner: "@mongodb/product-perf"
+Owner: Product Performance
+SlackSupportChannelName: "#performance"
+SlackSupportChannelId: C0V3KSB52
```

Comment:
Question: Could  `SlackSupportChannelId` ,  `SlackSupportChannelName` be stored in the mothra  teams meta data? That way it only needs to be changed / stored in one place.

If it ever changed then updating it just involves chaning the mothra repos teams data rather than 30 to 40 workload files.

Reply:
I would be inclined to do it that way ... I can see how I would know about the performance channel name but it might be more difficult to know where the C0V3KSB52 came from (particularly for a new channel).